src/main.ts

```diff
@@ -35,13 +34,14 @@ async function getChangeSet(client: AxiosInstance, workspaceId: string) {
   const changeSetsUrl = `/api/public/v0/workspaces/${workspaceId}/change-sets`
 
   let changeSetId = core.getInput('changeSetId')
-  let createdChangeSet = false
-  if (changeSetId === 'create') {
-    core.startGroup('Creating change set ...')
-    const changeSetName = core.getInput('changeSetName')
-    changeSetId = (await client.post(changeSetsUrl, { changeSetName })).data
+  if (!changeSetId) {
```

Comment:
This flow is a bunch clearer now that you gt rid of the `create` thing and just had "not specified" mean create. Thanks!